### ISSUE TITLE
Fix translations

### ISSUE THREAD

**Issue by ghost** [seguri/Lock#1]
Lint errors on translations:

````
:app:lintVitalRelease/home/krt/src/fdroiddata/build/name.seguri.android.lock/app/src/main/res/values/strings.xml:2: Error: "app_name" is not translated in "it" (Italian) [MissingTranslation]
    <string name="app_name">Lock</string>
            ~~~~~~~~~~~~~~~

   Explanation for issues of type "MissingTranslation":
   If an application has more than one locale, then all the strings declared
   in one language should also be translated in all other languages.

   If the string should not be translated, you can add the attribute
   translatable="false" on the <string> element, or you can define all your
   non-translatable strings in a resource file called donottranslate.xml. Or,
   you can ignore the issue with a tools:ignore="MissingTranslation"
   attribute.

   By default this detector allows regions of a language to just provide a
   subset of the strings and fall back to the standard language strings. You
   can require all regions to provide a full translation by setting the
   environment variable ANDROID_LINT_COMPLETE_REGIONS.

   You can tell lint (and other tools) which language is the default language
   in your res/values/ folder by specifying tools:locale="languageCode" for
   the root <resources> element in your resource file. (The tools prefix
   refers to the namespace declaration http://schemas.android.com/tools.)

1 errors, 0 warnings
````

I can workaround that locally with `echo -e "android { lintOptions { disable 'MissingTranslation'\n\ndisable 'ExtraTranslation' } }" >> app/build.gradle`, but instead this should either be fixed on your side or workaround like this on your side as well.

**Comment by seguri** [comment]
Very strange. I fixed that in tag 1.0.1 with `translatable="false"`. For some unknown reason, I can't see the tag and that fix. I'll now fix this in a 1.1.1 version.

**Comment by ghost** [comment]
Builds fine now, thanks for the quick reply.
